cdap-cli/src/main/java/co/cask/cdap/cli/commandset/ScheduleCommands.java

```diff
@@ -121,9 +124,11 @@ public void perform(Arguments arguments, PrintStream printStream) throws Excepti
       }
 
       String appId = programIdParts[0];
-      String scheduleId = programIdParts[1];
-      scheduleClient.suspend(appId, scheduleId);
-      printStream.printf("Successfully suspended schedule '%s' in app '%s'", scheduleId, appId);
+      String scheduleName = programIdParts[1];
```

Comment:
Not necessary, but it makes it understandable what `programIdParts[1]` is without having to look up javadocs for `ScheduleClient`.